tests/app/questionnaire/test_path_finder.py

```diff
@@ -1011,3 +1011,33 @@ def test_remove_answer_and_block_if_routing_backwards(self):
 
         self.assertEqual(path_finder.completed_blocks, [completed_blocks[0]])
         self.assertEqual(len(path_finder.answer_store.answers), 1)
+
+    def test_route_based_on_answer_count(self):
+        schema = load_schema_from_params('test', 'routing_answer_count')
+
+        answer_group_id = 'first-name'
+        fname_answer = Answer(
```

Comment:
`fname_answer`? whats `fname`?

Reply:
we generally try to give proper variable names bit abbreviations.

Reply:
"bit abbreviations"?

Reply:
"not abbreviations." sorry